lib/phoenix/test/channel_test.ex

```diff
@@ -596,11 +597,13 @@ defmodule Phoenix.ChannelTest do
   defp match_topic_to_channel!(socket, topic) do
     unless socket.handler do
       raise """
-      No socket handler found to lookup channel for topic #{inspect topic}.
-      Use `connect/3` when calling `subscribe_and_join` without a channel, for example:
+      no socket handler found to lookup channel for topic #{inspect topic}.
+      Use connect/3 when calling subscribe_and_join/* (or subscribe_and_join!/*)
```

Comment:
In this case, the asterisks meant "all arities of that function". 

That syntax is supported by the `:locals_without_parens` option in [`Code.format_string!/2`](https://hexdocs.pm/elixir/Code.html#format_string!/2), so I thought people would understand it. In short words, I thought it would equivalent to:

``` 
Use connect/3 when calling subscribe_and_join/2, subscribe_and_join/3, or
subscribe_and_join/4 (or subscribe_and_join!/2, subscribe_and_join!/3, or subscribe_and_join!/4)